tests/umockdev.c

```diff
@@ -1110,6 +1135,37 @@ test_hotplug_add_remove(UMockdevTestbedFixture * fixture, UNUSED_DATA)
 	libusb_hotplug_deregister_callback(fixture->ctx, handle_remove);
 }
 
+static void
+test_hotplug_udev_not_ready(UMockdevTestbedFixture * fixture, UNUSED_DATA)
```

Comment:
Dunno, a test is nice, but then maybe having one that potentially breaks if umockdev is fixed isn't …

I opened https://github.com/martinpitt/umockdev/issues/267

Reply:
> Dunno, a test is nice, but then maybe having one that potentially breaks if umockdev is fixed isn't …
> 
> I opened https://github.com/martinpitt/umockdev/issues/267

Yeah... Probably it makes sense to keep it in another PR that lands when umockdev is fixed?